### ISSUE TITLE
Force use just one Direct chat between users

### ISSUE THREAD

**Issue by psaavedra** [vector-im/riot-web#3457]
### Description

Could be nice if thre is a mode where Direct chat are foreced to be done in the same Chatroom and a user can not create more than 1 Direct chat for each person. 

#### Suggested tags:
priority: P4: intereseting idea - bluesky some day
bug or feature: feature
bug severity: cosmetic - feature works functionally but UI/UX is broken
additional categories:
release blocker - no
ui/ux - yes
network - none
platform - any

### Steps to reproduce

Start a new chat with a particualr users always starts the same Chatroom (if the chatroom is not previously created, this is created at first time)

Currently, you can start many Direct chat with a same person. This difficult diving into the historical looking for old conversations.

**Comment by devnoname120** [comment]
Note: the current behavior has the advantage of enabling to have both an encrypted chat and a clear chat to the same person at a time.

**Comment by TheLastProject** [comment]
While I'd accept an "you already have a direct chat with this user, want to go there instead?" dialog box before creating a new one, I very much would NOT like this forced on me. I have several direct chats with one person to discuss several subjects. For example, YouTube links to music we throw in one chat, general discussions in another, so we can just have a proper general chat and check out music we like to share when we feel like it without it disturbing the main conversation.

**Comment by ara4n** [comment]
yup, having multiple DMs with one user (one encrypted, one not, one social, one work etc) is very much the desired behaviour. However, agreed that the current semantics are fairly muddled. /develop already implements a new "do you want to reuse an existing DM or start a new one?" dialog when you hit "start chat" which mitigates the concerns here fairly well...

**Comment by jerrykan** [comment]
I support an internal matrix server which we run at work and I have noticed quite a few times where users unintentionally creating extra DM channels and then getting confused. Adding a dialog box to prompt users to reuse existing channels would be a big improvement from a usability perspective.

**Comment by turt2live** [comment]
Worth noting that if you use the 'Start Chat' button in the bottom right you do get a dialog. If you use the member info (right hand side after clicking someone's avatar) 'Start Chat' button it creates a new one. There's supposed to be a list of existing rooms above that button.

**Comment by jerrykan** [comment]
@turt2live Ah, interesting. I didn't realise the dialog box functionality already existed to some extend, I've only ever used the 'Start Chat' button (and it seems most of our users at work are the same). Discussing this with a few people at work, none of us noticed the existing rooms listed above the 'Start Chat' button until it was pointed out to us.

Could I suggest that the dialog box functionality be added to the 'Start Chat' button?

I am guessing that most of the time people want to continue talking in an existing direct chat channel rather than starting a new one... and for the infrequent times that users do really want to create a new channel the dialog box would just be a good confirmation.

**Comment by t3chguy** [comment]
Related https://github.com/vector-im/riot-web/issues/11495
